core/domain/learner_playlist_services_test.py

```diff
@@ -100,162 +101,128 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_adding_subscribed_exploration_to_playlist_does_not_add_again(self):
+        # Subscribe to exploration.
+        subscription_services.subscribe_to_exploration(
+            self.user_id, 'SUBSCRIBED_ID')
+
+        # Now if we try to add the same exploration, it shouldn't be added.
+        learner_progress_services.add_exp_to_learner_playlist(
+            self.user_id, 'SUBSCRIBED_ID')
         self.assertEqual(
             self._get_all_learner_playlist_exp_ids(self.user_id), [])
 
-        # Add an exploration to the learner playlist.
+        learner_playlist_services.remove_exploration_from_learner_playlist(
+            self.user_id, 'SUBSCRIBED_ID')
+
+        # Test adding the exploration_id if it is already in
+        # learner_playlist.exploration_ids
+        # Add two explorations to the learner playlist.
         learner_progress_services.add_exp_to_learner_playlist(
             self.user_id, self.EXP_ID_0)
-        self.assertEqual(
```

Comment:
Why is this part removed?

Reply:
I didn't think it was necessary to assert here since I assert in line 125 for both EXP_ID_0 and EXP_ID_1.

Reply:
Got it!